src/layouts/UserLayout.less

```diff
@@ -1,15 +1,20 @@
 @import "~antd/lib/style/themes/default.less";
 
-.container {
-  background: #f0f2f5;
-  background-image: url('https://gw.alipayobjects.com/zos/rmsportal/TVYTbAXWheQpRcWDaDMu.svg');
-  width: 100%;
-  min-height: 100%;
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 100%;
-  padding: 110px 0 144px 0;
-  position: relative;
+.content {
+  padding: 32px 0;
+}
+
+@media (min-width: @screen-md-min) {
+  .container {
+    background-image: url('https://gw.alipayobjects.com/zos/rmsportal/TVYTbAXWheQpRcWDaDMu.svg');
+    background-repeat: no-repeat;
+    background-position: center 110px;
+    background-size: 100%;
+  }
+
+  .content {
+    padding: 110px 0 144px 0;
```

Comment:
The padding-bottom(144px) is too large, because the current padding-bottom covers the footer content(which is absolute position), in this pr the dom has changed.

<img width="1183" alt="2018-01-17 6 25 32" src="https://user-images.githubusercontent.com/7017767/35038259-39f71b6e-fbb5-11e7-8607-90715b8be66d.png">

Reply:
just let me know which padding fits better and i update it,  footer is approx 50px so around 100?

maybe change 110 0 144px  0   to  8+8*11  =   padding: 96px 0; ??

Reply:
`padding: 110px(or 112px) 0 24px 0` is ok